class MappingException(Exception):
    """Custom exception raised during test data mapping."""
    pass
